### ISSUE TITLE
Wiki: How to install bourbon and neat in ember-cli

### ISSUE THREAD

**Issue by ahmed1490** [chancancode/hn-reader#10]
Can we have a wiki on what did you do to install bourbon and neat?

Looks like you did some unconventional stuff!

**Comment by chancancode** [comment]
I basically copied https://github.com/yapplabs/ember-cli-bourbon, except I tried to be a hipster and use the latest version (which requires latest sass, which requires Ruby sass instead of libsass, which turned out to be a bad idea – see #5 :smile:). I ended up using few of bourbon/neat anyway, pretty sure I would have been fine with the older version from `ember-cli-bourbon` :smile:

To be honest, I still don't know if it really requires creating an in-repo-addon just to use bourbon/neat, if I'm just doing it for one local project. I couldn't figure out how to make the sass `@import` work without doing what yapplabs did. Maybe @stefanpenner will know :stuck_out_tongue:
